app/src/main/java/com/kickstarter/ui/activities/ProfileActivity.java

```diff
@@ -33,20 +32,18 @@
 import butterknife.ButterKnife;
 import rx.android.schedulers.AndroidSchedulers;
 
-import static com.kickstarter.libs.utils.IntegerUtils.isNonZero;
-
-@RequiresActivityViewModel(ProfileViewModel.class)
-public final class ProfileActivity extends BaseActivity<ProfileViewModel> {
+@RequiresActivityViewModel(ProfileViewModel.ViewModel.class)
+public final class ProfileActivity extends BaseActivity<ProfileViewModel.ViewModel> {
   private ProfileAdapter adapter;
   private RecyclerViewPaginator paginator;
 
-  protected @Bind(R.id.avatar) ImageView avatarImageView;
-  protected @Bind(R.id.user_name) TextView userNameTextView;
-  protected @Bind(R.id.created_num) TextView createdNumTextView;
-  protected @Bind(R.id.backed_num) TextView backedNumTextView;
-  protected @Bind(R.id.created) TextView createdTextView;
-  protected @Bind(R.id.backed) TextView backedTextView;
-  protected @Bind(R.id.divider) View dividerView;
+  protected @Bind(R.id.avatar_image_view) ImageView avatarImageView;
```

Comment:
nice nice, is this our style going forward? suffix to describe the UI component?